templates/Azure/Certificates/Request.yaml

```diff
@@ -143,10 +143,35 @@ steps:
         New-PACertificate $domain -Plugin Azure -PluginArgs $pArgs -Verbose
 
         $generatedCert=$(Get-PACertificate)
+        # chain.cer, chain0.cer and chain1.cer are outputted files, which chain1.cer contains only the intermediate
+        $intermediatePath=$($generatedCert.ChainFile -replace 'chain.cer','chain1.cer'
 
+        Write-Host "##vso[task.setvariable variable=certPath;isOutput=true]$($generatedCert.CertFile)"
+        Write-Host "##vso[task.setvariable variable=intermediatePath;isOutput=true]$intermediatePath"
+        Write-Host "##vso[task.setvariable variable=privateKeyPath;isOutput=true]$($generatedCert.KeyFile)"
         Write-Host "##vso[task.setvariable variable=pfxPath;isOutput=true]$($generatedCert.PfxFullChain)"
         Write-Host "##vso[task.setvariable variable=pfxPass;isOutput=true;issecret=true]$pfxPassword"
 
+  - bash: |
+      outputPfx="$(Build.SourcesDirectory)/output.pfx"
+      outputPem="$(Build.SourcesDirectory)/chainCert.pem"
+
+      echo "" > ${outputPem}
+      cat ${certFile} ${intermediateFile} ${privateKeyFile} > ${outputPem}
```

Comment:
Don't think you need the private key file here, just the server and intermediate. Although I assume it'd still work ok